src/main/webapp/app/exam/exam-scores/exam-scores-average-scores-graph.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="row">
+    <canvas
+        class="col-md-7"
+        baseChart
+        [height]="height"
+        [datasets]="chartData"
+        [labels]="barChartLabels"
+        [options]="barChartOptions"
+        [legend]="chartLegend"
+        [chartType]="barChartType"
+    ></canvas>
```

Comment:
```suggestion
    <div class="col-md-7">
        <canvas
            baseChart
            [height]="height"
            [datasets]="chartData"
            [labels]="barChartLabels"
            [options]="barChartOptions"
            [legend]="chartLegend"
            [chartType]="barChartType"
        ></canvas>
    </div>
```

That should fix the issue.